azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -16,6 +16,18 @@
 
 package azkaban.project;
 
+import azkaban.database.AbstractJdbcLoader;
```

Comment:
Oops. Actually I'm refactoring this class. Eventually I will use another class (`JdbcProjectImpl`) to replace this class. Of course, you can still use this. But I would probably change `AzkabanCommonModule` to allow `JdbcProjectImpl` to be the only one implmentation of `Projectloader`.